libs/common-ui/src/lib/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,10 @@
+type Props = {
+    type: string;
+    name: string;
+    className?:string;
+    placeholder?: string;
+  };
+  
+  export const Input = ({ type, name, className, placeholder }: Props) => {
+    return <input className={className} type={type} name={name} placeholder={placeholder}/>;
```

Comment:
Because we're using tailwind it's safer to use [tailwind-merge](https://www.npmjs.com/package/tailwind-merge) before we place any css class to html element. Wrong order of Tailwind classes could make conflicts and strange behavior ;) 

More over you can make more extensible Input component using `ComponentProps` type (`import { type ComponentProps } from 'react';`)